Game/SexEngine/SexActivity/SexVaginalOnAllFours.gd

```diff
@@ -808,6 +816,13 @@ func getSubOrgasmHandlePriority():
 		return 10
 	return 5
 
+func fetishesToMod():
```

Comment:
This is too much for this pull request. Please remove the fetish modifying, way outside of the scope of a pull request that adds a new skill